src/coreComponents/python/modules/pygeosx_tools_package/pygeosx_tools/file_io.py

```diff
@@ -0,0 +1,53 @@
+
+import os
+import numpy as np
+
+
+def save_tables(grid, properties, table_root='./tables', axes=['x', 'y', 'z', 't']):
+    """
+    Saves a set of tables in GEOSX format
+    Notes: The shape of these arrays should match the length of each axis in the specified order
+           The output directory will be created if it does not exist yet
+
+    Args:
+        grid (list): A list of numpy ndarrays defining the table axes
+        properties (dict): A dict of numpy ndarrays defning the table values
+        table_root (str): The root path for the output directory
+        axes (list): A list of names for each potential axis (default = ['x', 'y', 'z', 't'])
+    """
+    os.makedirs(table_root, exist_ok=True)
+    for ii, d in zip(range(len(grid)), axes):
```

Comment:
Are you OK if `grid` and `axes` do not share the same length?

Reply:
I've added some checks to verify that the grid, property sizes are consistent.  I've also added some additional code to handle the default axes names for various lengths:

- N=1, axes=t
- N=3, axes=x, y, z
- N=4, axes=x, y, z, t
- Otherwise, x0, x1, x2, x3, x4, ....